### DISCUSSION TITLE
NET_LOCAL Multiple Arguments

### DISCUSSION THREAD

**Discussion by polyboi** [bubuntux/nordlynx#147] [General]
Hi,

I'm trying to connect remotely using a wireguard tunnel to my router to apps that run in containers that connect through my Nordlynx container but when I add the subnet to NET_LOCAL it doesn't seem to work. Let's say for example my LAN is on 192.168.1.0/24 and my wireguard interface on my router is on 192.168.2.0/24 I thought I could just add that to NET_LOCAL and it would treat my remote connection like any other on the LAN. I added it to my docker-compose.yml like this...

- NET_LOCAL =192.168.1.0/24,192.168.2.0/24

However when I tried that nothing seems to work. I can't connect through my remote wireguard connection through my router and I can't connect on my LAN. I can get it to work if I just use a single subnet that includes all the IPs I need ie.

-NET_LOCAL=192.168.0.0/16

However I was curious to know why the first option didn't work and I feel like it might be a bit more secure to lock down access to two smaller subnets, ie. my LAN and possibly only a single IP on my remote connection. Something like...

- NET_LOCAL=192.168.1.0/24,192.168.2.10/32

I hope this makes sense. When it comes to networking, firewalls and subnets etc I feel like I'm just feeling around in the dark trying to push every button until I find one that works. Any help would be appreciated.

**Comment by DuncanTheFox** [comment]
Hi Polyboi,

Not sure if you ever found the issue. But I tested it, locally it was working with 192.168 for me. But not when using Wireguard to connect to my home network. I added 192.168.1.0/24 192.168.2.10/32 to the config as NET_LOCAL, it seems to be SPACE separated of all separators. This works for me.

Kind regards,

Duncan.
